Tasks/Gradle/CodeAnalysis/Common/AnalysisResult.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Data class that describes code analysis results
+ * 
+ * @export
+ * @class AnalysisResult
+ */
+export class AnalysisResult {
+    constructor(public toolName: string, public moduleName: string, public resultFiles: string[], public violationCount: number, public affectedFileCount: number) {
```

Comment:
the args become public properties when giving them the public visibility in the ctor (syntactic sugar)